### PR TITLE
New questions to Github Actions

### PR DESCRIPTION
**Pull request by dostonhamrakulov**
## PR Type

- [x] Adding new question(s)
> **Warning**: We do not support the inclusion of questions directly copied from official GitHub certification exams. Please only submit original questions and content that you have created.
- [ ] Other content changes (updating questions, answers, explanations or study resources)
- [ ] Code changes (non-content)
- [ ] Documentation changes
- [ ] Other

## Related issue(s)

https://github.com/FidelusAleksander/ghcertified/issues/117

### CODE REVIEW

**Comment by dostonhamrakulov** [comment]
Hi @FidelusAleksander ,
I am planning to take GitHub Actions Certification exam and learning it intensively, I hope if you don't mind if I bring more questions and topics here as part of my learning and contributing to this great project

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]

**Comment by FidelusAleksander** [comment]
> Hi @FidelusAleksander ,
> I am planning to take GitHub Actions Certification exam and learning it intensively, I hope that you don't mind if I bring more questions and topics here as part of my learning and contributing to this great project

Go ahead! That's the whole idea behind the project 😎🚀

**Comment by FidelusAleksander** [comment]
> Hi @FidelusAleksander ,
> I am planning to take GitHub Actions Certification exam and learning it intensively, I hope that you don't mind if I bring more questions and topics here as part of my learning and contributing to this great project

Go ahead! That's the whole idea behind the project 😎🚀

File: content/questions/actions/question-096.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+question: "Select a default environment variable that contains the operating system of the runner executing the job?"
```
[the remark this answers is not in the document]
Comment (dostonhamrakulov) [reply content/questions/actions/question-096.md:2]: Updated

File: content/questions/actions/question-097.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+question: "How does the cache action in GitHub Actions handle a cache miss?"
```
[the remark this answers is not in the document]
Comment (dostonhamrakulov) [reply content/questions/actions/question-097.md:2]: Updated

File: content/questions/actions/question-098.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+archetype: "questions"
+title: "Question 098"
+draft: false
+---
+
+> https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+1. [ ] add a condition in the workflow YAML for weekdays
+1. [ ] it is not possible in GitHub actions
+1. [ ] utilize the on: schedule: weekdays configuration
+1. [x] use the on: schedule: cron syntax
```
[the remark this answers is not in the document]
Comment (dostonhamrakulov) [reply content/questions/actions/question-098.md:12]: updated

File: content/questions/actions/question-101.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+---
+question: "How do you ensure that `Upload Failure test report` step is be executed only if `Run Tests` step fails?"
+archetype: "questions"
+title: "Question 101"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions
+1. [x]
```
[the remark this answers is not in the document]
Comment (dostonhamrakulov) [reply content/questions/actions/question-101.md:11]: Thanks, I have fixed it:

**Review by FidelusAleksander — APPROVED** [review]
This looks good to merge :+1: 

Thanks for contributing @dostonhamrakulov  :rocket:  !

**Pull request merged**